ccdImports/configFiles/CCD_Probate_Backoffice/CaseEventToFields.json

```diff
@@ -56,7 +56,7 @@
   {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "CaseEventID": "solicitorUpdateProbate", "CaseFieldID": "dispenseWithNoticeOverview", "PageFieldDisplayOrder": 3, "DisplayContext": "MANDATORY", "PageID": "solicitorUpdateProbatePage3", "PageDisplayOrder": 3, "FieldShowCondition": "dispenseWithNoticeLeaveGiven=\"No\"", "ShowSummaryChangeOption": "y"},
   {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "CaseEventID": "solicitorUpdateProbate", "CaseFieldID": "dispenseWithNoticeSupportingDocs", "PageFieldDisplayOrder": 4, "DisplayContext": "MANDATORY", "PageID": "solicitorUpdateProbatePage3", "PageDisplayOrder": 3, "FieldShowCondition": "dispenseWithNoticeLeaveGiven=\"No\"", "ShowSummaryChangeOption": "y"},
   {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "CaseEventID": "solicitorUpdateProbate", "CaseFieldID": "dispenseWithNoticeOtherExecsList", "PageFieldDisplayOrder": 5, "DisplayContext": "MANDATORY", "PageID": "solicitorUpdateProbatePage3", "PageDisplayOrder": 3, "ShowSummaryChangeOption": "y"},
-  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "CaseEventID": "solicitorUpdateProbate", "CaseFieldID": "solsExecutorInfoSection", "PageFieldDisplayOrder": 1, "DisplayContext": "READONLY", "PageID": "solicitorUpdateProbatePage4", "PageLabel": "About the executors", "PageDisplayOrder": 4, "ShowSummaryChangeOption": "n"},
+  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "CaseEventID": "solicitorUpdateProbate", "CaseFieldID": "solsExecutorInfoSection", "PageFieldDisplayOrder": 1, "DisplayContext": "READONLY", "PageID": "solicitorUpdateProbatePage4", "PageLabel": "About the executors", "PageDisplayOrder": 4, "ShowSummaryChangeOption": "n", "CallBackURLMidEvent": "http://${CCD_DEF_CASE_SERVICE_BASE_URL}/case/sols-validate-executors"},
```

Comment:
just checking its ok for the scheme to be in url? Http? Would it not be https for prod?